system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.cpp

```diff
@@ -104,7 +122,7 @@ std::string PeriodicMeasurementNode::GetStatusString() const
   std::stringstream ss;
   ss << "name=" << get_name() <<
     ", measurement_period=" << std::to_string(measurement_period_.count()) << "ms" <<
-    ", publishing_topic=" << publishing_topic_ <<
+    ", publishing_topic=" << ((publisher_ == nullptr) ? "" : publisher_->get_topic_name()) <<
```

Comment:
The .hpp still has ```publishing_topic_```. Why tie displaying the topic to the node state?

Reply:
If displaying the topic doesn't depend on the node state, then when the user does topic name remapping, the output is technically wrong.

Reply:
Could this be simplified to `publisher_ ? publisher_->get_topic_name() : ""`

Reply:
Done.